### PR TITLE
rns.online

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/rns.online.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<ruleset name="rns.online">
+	<target host="rns.online" />
+	<target host="www.rns.online" />
+	<target host="cdn.rns.online" />
+	<test url="http://cdn.rns.online" />
+	<test url="http://cdn.rns.online/" />
+	<test url="http://cdn.rns.online?" />
+	<test url="http://cdn.rns.online#" />
+	<test url="http://cdn.rns.online/?" />
+	<exclusion pattern="^http://cdn\.rns\.online/?(?:$|\?|#)" />
```
Comment (J0WI) [inline src/chrome/content/rules/rns.online.xml]: A trailing slash is required.
Please also avoid non-capturing groups, as our code style recommends.

**Comment by galeksandrp** [comment]
Improved.

**Comment by J0WI** [comment]
There are still test urls without trailing slashes. That's wrong, because the `?` and `#` chars are not part of the TLD.
Please also remove the change in translations.

**Comment by galeksandrp** [comment]
Improved.

File: src/chrome/content/rules/rns.online.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	<target host="m.rns.online" />
+	<test url="http://cdn.rns.online/" />
+	<test url="http://cdn.rns.online/?" />
+	<test url="http://cdn.rns.online/?q" />
+	<test url="http://cdn.rns.online/?s" />
+	<test url="http://cdn.rns.online/#" />
+	<test url="http://cdn.rns.online/#q" />
+	<exclusion pattern="^http://cdn\.rns\.online/?$" />
+	<exclusion pattern="^http://cdn\.rns\.online/?\?" />
+	<exclusion pattern="^http://cdn\.rns\.online/?#" />
```
Comment (J0WI) [inline src/chrome/content/rules/rns.online.xml]: `?` need to be escaped or removed. Exclusions without trailing slash are not allowed.

Comment (galeksandrp) [inline src/chrome/content/rules/rns.online.xml]: Why?

Comment (J0WI) [inline src/chrome/content/rules/rns.online.xml]: https://stackoverflow.com/questions/1617058/ok-to-skip-slash-before-query-string#answer-1617074

Comment (galeksandrp) [inline src/chrome/content/rules/rns.online.xml]: Trailing slash forced ins exclusions.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**